packages/components/src/collection/src/getItemText.tsx

```diff
@@ -3,7 +3,7 @@ import { getRawSlots, isNil } from "../../shared";
 
 export function getItemText(item: CollectionNode): string {
     if (isDivider(item) || isItem(item)) {
-        const { stringValue, text } = getRawSlots(item?.content, ["text"]);
+        const { stringValue, text } = getRawSlots(<>{item?.content}</>, ["text"]);
```

Comment:
This is odd, i don't understand why we need to specify fragments.

Reply:
After investigation, I saw that the item.content can by either a ReactElement or a ElementType[].
getRawSlots receive a ReactNode

 ElementType is not part of ReactNode, so wrapping it makes it work, but the issue should be fixed

The typing here should be fixed another way

Reply:
Do you have any suggestions for a fix?

I'm not too familiar with ElementType, ReactElement, etc.